ansible/roles/cli/files/createContentJson.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Just copy the existing script from the tools folder to here so that Git history shows the file was moved.